apps/storefront/app/grunnleggende/designelementer/farger/ColorInfoTable.tsx

```diff
@@ -0,0 +1,27 @@
+'use client';
+import { colorMetadata } from '@digdir/designsystemet';
+import { Table } from '@digdir/designsystemet-react';
+
+export const ColorInfoTable = () => {
+  return (
+    <Table data-color='neutral' zebra border style={{ margin: '2rem 0' }}>
+      <Table.Head>
+        <Table.Row>
+          <Table.HeaderCell>Navn</Table.HeaderCell>
+          <Table.HeaderCell>Bruksområde</Table.HeaderCell>
+        </Table.Row>
+      </Table.Head>
+      <Table.Body>
+        {Object.keys(colorMetadata).map((key) => {
+          const item = colorMetadata[key as keyof typeof colorMetadata];
+          return (
+            <Table.Row key={key}>
+              <Table.Cell>{item.number + '. ' + item.name}</Table.Cell>
+              <Table.Cell>{item.description.short}</Table.Cell>
+            </Table.Row>
+          );
+        })}
```

Comment:
Suggest using `Object.values` (or `.entries`) for mapping colors to avoid casting types.

`colorMetadata` `key` and `item.name` are the same so you can just use the `.values`

```suggestion
        {Object.values(colorMetadata).map((item) => (
          <Table.Row key={item.name}>
            <Table.Cell>{item.number + '. ' + item.name}</Table.Cell>
            <Table.Cell>{item.description.short}</Table.Cell>
          </Table.Row>
        ))}
```